plugins/linux-v4l2/v4l2-output.c

```diff
@@ -98,6 +99,12 @@ static int loopback_module_load()
 	return run_command(
 		"pkexec modprobe v4l2loopback exclusive_caps=1 card_label='OBS Virtual Camera' && sleep 0.5");
 }
+#else
```

Comment:
This should be restricted to the FreeBSD platform.

Reply:
Could we have a generic solution here that scans all /dev/video<N> devices instead? Non-linux systems does not have any of the commands that the code uses to detect the virtual webcam.

Reply:
No that wouldnt be appropriate for linux systems where we want to offer the ability to load the module on demand.